apps/language_models/src/pipelines/falcon_pipeline.py

```diff
@@ -482,11 +482,16 @@ def generate_new_token(self):
     default_prompt_text = "Girafatron is obsessed with giraffes, the most glorious animal on the face of this Earth. Giraftron believes all other animals are irrelevant when compared to the glorious majesty of the giraffe.\nDaniel: Hello, Girafatron!\nGirafatron:"
     continue_execution = True
 
+    print("\n-----\nScript executing for the following config: \n")
+    print("Falcon Model: ", falcon.model_name)
+    print("Precision:    ", args.precision)
+    print("Device:       ", args.device)
+
     while continue_execution:
         use_default_prompt = input(
             "\nDo you wish to use the default prompt text? True or False?: "
```

Comment:
Instead just ask for `Y/N` (case-insensitive) just like what we're prompted during any pip/sudo apt-get/etc installations.